### PR TITLE
Home task2

### PR DESCRIPTION
**Pull request by Nadya1993**
Postman collection: https://www.getpostman.com/collections/6211e69d33fdedede493

### CODE REVIEW

File: utils/generateUsers.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  login: string;
+  password: string;
+  age: number;
+}
+
+export type User = GeneralProperties & {
+  isDeleted: boolean;
+}
+
+export class UserManager {
```
Comment (spiderpoul) [inline utils/generateUsers.ts]: it's more like `model`, i suggest to put in separate file in `models/UserModel`

File: utils/validation.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import Joi from 'joi';
```
Comment (spiderpoul) [inline utils/validation.ts]: I suggest to create a separate file for user validation, f.e. `userValidation`

File: src/routes/index.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
+import express from 'express';
+import { UserManager } from '../../utils/generateUsers';
+import { UserRes } from '../types';
+import { schema, reduceErrorResponse } from '../../utils/validation';
+
+const router = express.Router();
+const userManager = new UserManager(5);
+
+router.get('/users', (req: express.Request, res: express.Response) => {
```
Comment (spiderpoul) [inline src/routes/index.ts]: it's better to create a separate file for user routes, f.e. `userRoutes`

File: utils/generateUsers.ts (hunk 1 above)
Comment (Nadya1993) [reply utils/generateUsers.ts]: Fixed

File: src/routes/index.ts (hunk 3 above)
Comment (Nadya1993) [reply src/routes/index.ts]: Fixed

File: utils/validation.ts (hunk 2 above)
Comment (Nadya1993) [reply utils/validation.ts]: Fixed

**Pull request merged**